extensions/shared/get-validated-attributes.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * External dependencies
+ */
+import { reduce } from 'lodash';
+
+export const getValidatedAttributes = ( attributeDetails, attributesToValidate ) =>
```

Comment:
Could you add a docblock with as much information as possible about how this can be used, what must be passed to it, and what it will return? This would allow others to use that function with too much trouble.